src/utils/upload/index.ts

```diff
@@ -49,6 +76,33 @@ export function areFilesSame(
     }
 }
 
+export function fileHashExists(file: Metadata) {
+    if (file.hash || (file.imageHash && file.videoHash)) {
```

Comment:
we can directly return this expression as its a boolean expression 
i.e., 
`return file.hash || (file.imageHash && file.videoHash)`